src/components/Card/Card.js

```diff
@@ -28,10 +33,10 @@ export function Card({ image, name, temperament, className, mode, id }) {
 }
 
 Card.propTypes = {
-  image: PropTypes.object.isRequired,
+  image: PropTypes.object,
   name: PropTypes.string.isRequired,
   temperament: PropTypes.string,
   className: PropTypes.string,
-  mode: PropTypes.oneOf(['default', 'primary']),
+  mode: PropTypes.oneOf(["default", "primary"]),
```

Comment:
default", "primary" - not very clear names of the property values

if you have a default, it is unlikely that you need to use the `primary`, it is better to describe in the name of the alternative mode how it differs from the default mode